tests/utils/_withPageAndStaticServer.js

```diff
@@ -43,6 +45,19 @@ module.exports = async (t, run) => {
   try {
     // Wait for no more network requests for at least 500ms before passing onto main test
     await page.goto(t.context.staticUrl, {waitUntil: 'networkidle0'});
+
+    // Expose safeEvaluate function to try...catch page functions for better stack traces
+    await page.evaluate(() => {
+      window.safeEvaluate = async function(fn, ...args) {
```

Comment:
Why do we need to take `args` here?

If the caller really wanted to pass in args they can capture a closure, or use `Function.bind`. Is there another argument for why it's the business of `safeEvaluate` to manage the caller's arguments for them?

Reply:
yeah you're right! `safeEvaluate` shouldn't really need the args, I've got rid of them!